packages/test/test-end-to-end-tests/src/test/rootDatastores.spec.ts

```diff
@@ -394,43 +391,29 @@ describeFullCompat("Named root data stores", (getTestObjectProvider) => {
 			},
 		);
 
-		/**
-		 * Aliasing datastores summarized before the alias op is sent and after the attach op is sent
-		 * does not cause a datastore corruption issue
-		 *
-		 * This test validates a bug where the rootiness of a datastore was not set to true in the
-		 * above scenario.
-		 */
-		it("Aliasing a bound datastore marks it as root correctly", async () => {
-			const aliasableDataStore1 = await runtimeOf(dataObject1).createDataStore(packageName);
-			const aliasedDataStoreResponse1 = await aliasableDataStore1.request({ url: "/" });
-			const aliasedDataStore1 = aliasedDataStoreResponse1.value as ITestFluidObject;
-			// Casting any to repro a race condition where bindToContext is called before summarization,
-			// but aliasing happens afterwards
-			(
-				aliasableDataStore1 as IDataStoreWithBindToContext_Deprecated
-			).fluidDataStoreChannel?.bindToContext?.();
-			await provider.ensureSynchronized();
+		it("getAliasedDataStoreEntryPoint only returns aliased data stores", async () => {
+			const dataStore = await runtimeOf(dataObject1).createDataStore(packageName);
+			const dataObject = (await dataStore.entryPoint?.get()) as ITestFluidObject;
+			assert(dataObject !== undefined, "could not create data store");
 
-			let dataStore: FluidObject | undefined;
+			let aliasedDataStore: FluidObject | undefined;
 			let error: unknown;
 			try {
-				dataStore = await getAliasedDataStoreEntryPoint(
-					dataObject2,
-					aliasedDataStore1.runtime.id,
+				aliasedDataStore = await getAliasedDataStoreEntryPoint(
```

Comment:
nit: Rename this to say something about back compat?  My eyes didn't realize it was a helper that wrapped the old thing if the new thing was missing

Reply:
Updated the `getAliasedDataStoreEntryPoint` to talk about back compat.